tests/test_crs/valid_flent.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: flent-benchmark
+  namespace: my-ripsaw
+spec:
```

The CI is failing because indexing is not enabled in the test:
```suggestion
spec:
  elasticsearch:
    url: ES_SERVER
    index_name: ripsaw-flent
  metadata:
    collection: true
```